### PR TITLE
ci(kitchen+travis): add kitchen tests

### PR DESCRIPTION
**Pull request by n-rodriguez**

### CODE REVIEW

**Comment by myii** [comment]
@n-rodriguez Tests are failing at the moment.  And is it worth using the "tree"-shaped matrix from `template-formula`?

File: test/integration/default/inspec.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: locale
+title: Locale Formula
+maintainer: Nicolas Rodriguez
```
Comment (daks) [inline test/integration/default/inspec.yml]: don't you think 'Saltstack Formulas' is better suited?

Comment (n-rodriguez) [reply test/integration/default/inspec.yml]: for the maintainer?

Comment (daks) [reply test/integration/default/inspec.yml]: yes, as the community is the maintainer of the formulas, it seems logical for me

File: .gitignore [hunk 2]
```diff
@@ -1 +1,110 @@
...
+/site
+
+# mypy
+.mypy_cache/
+
+# Bundler
+Gemfile.lock
+
+# copied `.md` files used for conversion to `.rst` using `m2r`
+docs/*.md
```
Comment (myii) [inline .gitignore]:
```suggestion
docs/*.md

# Vim
*.sw?
```

**Comment by myii** [comment]
OK, that was a powerful response!  Since this is now `semantic-release`, I'll need to run a few more checks and get back to you.

**Comment by myii** [comment]
@n-rodriguez Merged, how many is that now?  Fantastic work, thanks.  @daks You're falling behind!!

**Pull request merged**